apps/namadillo/src/App/Staking/IncrementBonding.tsx

```diff
@@ -104,6 +105,10 @@ const IncrementBonding = (): JSX.Element => {
     seed: seed.current,
   });
 
+  const topValidatorsByRank = useMemo(() => {
```

Comment:
I moved this up one level b/c previously it was using `filteredValidators` so all of the sudden in the `inactive` view those validators were "topValidatorsByRank"